tests/test_speedify.py

```diff
@@ -198,14 +204,15 @@ def test_connect(self):
 
     def test_connect_country(self):
         logging.debug("\n\nTesting connect country...")
-        serverinfo = speedify.connect_country("sg")
+        country = random.choice(list(server_countries()))
+        serverinfo = speedify.connect_country(country)
```

Comment:
We want to test actually connecting to servers from multiple countries here too. `test_connectmethod` below is just testing changing the connect setting and not actually connecting to the servers, while this test is checking that it actually connects.